### ISSUE TITLE
Manuskript can't open my original project 

### ISSUE THREAD

**Issue by Oz-D** [olivierkes/manuskript#811]
Unfortunately I'm pretty sure my file has been lost forever. On 10/21, I had Manuskript opened but I had to turn it off as I was heading for work. I don't recall making much progress anyways so it wouldn't matter if I couldn't get that day's progress back.

But after trying to open the same .msk file again, it crashes. Opens, says debugging, version the typical stuff and then boop gone.
I've tried using my previous backups I've uploaded onto Drive but I facepalmed so hard when I realized I didn't backup the .msk file. 

I googled the issue and tried opening the .msk file in Text Editor and inserting 1 which fixed someone else's problem identical to mine and it opened!

Except, none of my work was there. It was completely empty and honestly, I don't know what to do anymore. I checked characters, the root, plots etc and nothing- it was a ghost town.

Now I'm contemplating switching from my favorite writing software to something else less risky but I'm hoping someone could wave a magic wand on this issue.

**Comment by Oz-D** [comment]
Is there some way I can get even a fraction of the work/notes I had on my novel?

**Comment by obw** [comment]
You can try to rename it to `project.zip` and extract the content. Perhaps the repair option of ZIP will also help.

the MSK file is nothing more then a ZIP-File with the folder-structure of a manuscript Project.

**Comment by Oz-D** [comment]
WinRAR tells me it's either in a unknown format or damaged, basically I can't open it when I rename the .msk file to .zip

**Comment by obw** [comment]
WinRar has a repair function under Tools try this, good luck.

I make Backups mostly daily, also I don't use the single File Project, I use the multi File setting, which has also the benefit that a Versioning.System can be used. (My Style of Backup), also so it's easy to roll-back when something does not work in the Story as planned!

**Comment by Oz-D** [comment]
Yeah it looks like I'll have to start doing non-stop backups if I ever want to use manuskript again.
Unfortunately when I repaired it and when it gave me a rebuilt_project.zip it had nothing in it. No files to extract :(
It seems like my first project ever has been killed before it could even be finished.

**Comment by super-temp** [comment]
I'm having the same issue. The fixes mentioned haven't worked for me. I also tried restoring from earlier versions on my Dropbox, but only the empty one that Dropbox saved pretty much the minute I created the file will open. None of the rest will.

**Comment by Oz-D** [comment]
Yeah it's a shame, I liked manuskript I think I'm just gonna have to settle with something less for now

**Comment by siliconserf** [comment]
Look in your trash folder and see if you can find any files that relate to the project. Forward what you recover and anything else. I may not be able to help, but it is an interesting challenge.
